src/stores/cardStore.spec.ts

```diff
@@ -0,0 +1,135 @@
+    expect(drawnCard).toHaveProperty("flipped");
+  });
+
+  it("should give player a card", () => {
+    const cardStore = new CardStore();
+    cardStore.givePlayerACard();
+
+    expect(cardStore.playerCards).toHaveLength(1);
+  });
+
+  it("should give banker a card", () => {
+    const cardStore = new CardStore();
+    cardStore.giveBankerACard();
+
+    expect(cardStore.bankerCards).toHaveLength(1);
+  });
+
+  it("should reset player cards", () => {
+    const cardStore = new CardStore();
+    cardStore.resetPlayerCards();
+
+    expect(cardStore.playerCards).toHaveLength(0);
+  });
+
+  it("should reset banker cards", () => {
+    const cardStore = new CardStore();
+    cardStore.resetBankerCards();
+
+    expect(cardStore.bankerCards).toHaveLength(0);
+  });
+});
+
+describe("should count points correctly", () => {
+  const cardStore = new CardStore();
+  cardStore.playerCards.push(fakeCards[0]); // 2
+  cardStore.playerCards.push(fakeCards[1]); // 9
+  cardStore.bankerCards.push(fakeCards[2]); // 0
+  cardStore.bankerCards.push(fakeCards[3]); // 3
+
+  it("should count player's points correctly", () => {
+    expect(cardStore.playerPoints).toEqual(1);
+  });
+
+  it("should count banker's points correctly", () => {
+    expect(cardStore.bankerPoints).toEqual(3);
+  });
+});
+
+describe("should flip cards correctly", () => {
+  const cardStore = new CardStore();
+
+  cardStore.playerCards.push(fakeCards[1]);
+  cardStore.playerCards.push(fakeCards[0]);
+
+  cardStore.bankerCards.push(fakeCards[3]);
+  cardStore.bankerCards.push(fakeCards[2]);
+
+  it("should flip player's cards", () => {
+    expect(cardStore.playerCards[0].flipped).toEqual(false);
+    expect(cardStore.playerCards[1].flipped).toEqual(false);
+
+    cardStore.flipPlayerCards();
+
+    expect(cardStore.playerCards[0].flipped).toEqual(true);
+    expect(cardStore.playerCards[1].flipped).toEqual(true);
+  });
+
+  it("should flip bankers's cards", () => {
+    expect(cardStore.bankerCards[0].flipped).toEqual(false);
+    expect(cardStore.bankerCards[1].flipped).toEqual(false);
+
+    cardStore.flipBankerCards();
+
+    expect(cardStore.bankerCards[0].flipped).toEqual(true);
+    expect(cardStore.bankerCards[1].flipped).toEqual(true);
+  });
+
+  it("should flip players's third card", () => {
+    cardStore.playerCards.push(fakeCards[3]);
+    expect(cardStore.playerCards[2].flipped).toEqual(false);
+    cardStore.flipThirdPlayerCard();
+    expect(cardStore.playerCards[2].flipped).toEqual(true);
+  });
+
+  it("should flip banker's third card", () => {
+    cardStore.bankerCards.push(fakeCards[3]);
+    expect(cardStore.bankerCards[2].flipped).toEqual(false);
+    cardStore.flipThirdBankerCard();
+    expect(cardStore.bankerCards[2].flipped).toEqual(true);
+  });
+});
+
+describe("should judge if next round is possible correctly", () => {
+  it("should return true", () => {
+    const cardStore = new CardStore();
+
+    expect(cardStore.nextRoundIsPossible()).toBeTruthy();
```

Comment:
`toBeTruthy()`  is okay to use but also please read this https://stackoverflow.com/a/32615835 🙂